.github/workflows/storybook-tests.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Storybook Tests'
```

Comment:
it would be best if we could move this under CI, because the triggers are the same, and because we want to use the existing required check to make sure that no PRs can be merged if the storybook tests don't pass. I think it should be pretty easy to do, just move the relevant steps (with the exception of the duplicated ones) to the ci workflow

Reply:
Makes sense. Resolved in [e78111f](https://github.com/nearform/quantum/pull/418/commits/e78111f7bde6d2d7b3139fa6d02954abd1308fe1).